src/main/java/com/rabbitmq/client/ExchangeType.java

```diff
@@ -0,0 +1,19 @@
+package com.rabbitmq.client;
+
+/**
+ *
+ */
+public enum ExchangeType {
```

Comment:
I suggest that we rename this to `BuiltinExchangeType` because exchange types are certainly not limited to these 4.